README.md

```diff
@@ -1,10 +1,14 @@
+<!-- TODO: Add badges for CI status and IBM Cloud deploy count -->
+
 # Use a deep learning model to filter images in a web application
 
-Every day 2.5 quintillion bytes of data are created, based on an [IBM study](https://public.dhe.ibm.com/common/ssi/ecm/wr/en/wrl12345usen/watson-customer-engagement-watson-marketing-wr-other-papers-and-reports-wrl12345usen-20170719.pdf). A lot of that data is unstructured data,
-such as large texts, audio recordings, and images. In order to do something useful
-with the data, we must first convert it to structured data.
+Every day 2.5 quintillion bytes of data are created, based on an
+[IBM study](https://public.dhe.ibm.com/common/ssi/ecm/wr/en/wrl12345usen/watson-customer-engagement-watson-marketing-wr-other-papers-and-reports-wrl12345usen-20170719.pdf).
+A lot of that data is unstructured data, such as large texts, audio recordings, and images. In order to do something
+useful with the data, we must first convert it to structured data.
 
-In this Code Pattern we will use one of the models from the [Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/),
+In this Code Pattern we will use one of the models from the
+[Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/),
 an exchange where developers can find and experiment with open source deep learning
```

Comment:
Should probably note here that MAX is the acronym. Like "Model Asset Exchange (MAX)"

Reply:
I'll add this